res/TensorFlowLiteRecipes/Part_Tanh_FC_nobias/test.recipe

```diff
@@ -0,0 +1,42 @@
+operand {
+  name: "in"
+  type: FLOAT32
+  shape { dim: 4 dim: 1536 }
+}
+operand {
+  name: "Tanh"
+  type: FLOAT32
+  shape { dim: 4 dim: 1536 }
+}
+operand {
+  name: "weight"
+  type: FLOAT32
+  shape { dim: 4362 dim: 1536 }
```

Comment:
Can we use `[4, 4]`? `{ dim: 4362 dim: 1536 }` seems to drop a hude model.

Reply:
I made recipe based on my local model. I will reduce the size if related issue can be reproduced.

Reply:
PR updated.

Reply:
@chunseoklee , somehow PR is not updated.

Reply:
Pushed again